### ISSUE TITLE
GPU memory leak in sevenn_infererence

### ISSUE THREAD

**Issue by YutackPark** [MDIL-SNU/SevenNet#187]
Can be solved with `.detach().cpu()`

**Comment by thangckt** [comment]
hi @YutackPark,

Does this relates to this issue #193 ?

**Comment by YutackPark** [comment]
@thangckt No it doesn't. I know the exact cause of this issue and it is irrelevant to LAMMPS. Please open relevant issue and chat there. I found your issue lacks information (you should give as much as information to help fixing the issue) for me to reproduce.
